java/core/src/test/java/sleeper/core/statestore/inmemory/InMemoryFileReferenceStoreTest.java

```diff
@@ -443,27 +442,6 @@ public void shouldSetFileReadyForGC() throws Exception {
                             assertThat(files).containsExactly("newFile"));
         }
```

Comment:
There's a potential that you might try to add a new file with multiple references, so we should decide whether to support that or not. If the partition for a compaction job has been split, it would make sense for the job to add multiple references to the new file on the new leaf partitions. We should probably have a test for that, in all the file reference store test classes. We can make that fail for now if it's easier.

Reply:
Thanks, added a test to all 3 statestores to fail when trying to add multiple references to a new file